apps/hash-graph/bench/util.rs

```diff
@@ -57,12 +57,52 @@ pub fn setup_subscriber(
         fn drop(&mut self) {}
     }
 
-    let mut target_dir = Path::new("out").join("tracing").join(group_id);
+    // Taken from Criterion source code
+    fn truncate_to_character_boundary(s: &mut String, max_len: usize) {
+        let mut boundary = cmp::min(max_len, s.len());
+        while !s.is_char_boundary(boundary) {
+            boundary -= 1;
+        }
+        s.truncate(boundary);
+    }
+
+    // Taken from Criterion source code
+    pub fn make_filename_safe(string: &str) -> String {
```

Comment:
I wonder if we want to move this function to `repo-chores` or another central place instead and don't rely on the internals of `criterion`

Reply:
Can do – no strong opinion

Reply:
I encountered an issue in a later PR using this method, did this in H-2705.